tests/assets/eks_service_role_policy.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+    ]
```

Comment:
Is this just a placeholder?

Reply:
Actually it was getting used earlier and I forgot to remove it. Have deleted the un needed jsons.